.github/workflows/agent-build.yml

```diff
@@ -0,0 +1,56 @@
+# The main "setup" workflow, that calls other workflows.
+name: Scalyr Agent Build.
+
+on: [push]
+
+# Agent image tests.
+jobs:
+  match-branches:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check branches.
```

Comment:
Will this step bill removed once this PR is merged into master?

Reply:
It shouldn't be removed, but we'll have to change the branch name to master, so it will call a re-usable workflow file from an appropriate branch.